src/cmds.js

```diff
@@ -29,17 +29,19 @@ const runCmd = (cmd, { errorMessage = 'Skipping... {0}' } = {}) => {
  * @param {*|string} version
  * @param {object} options
  * @param {string} options.contextPath
+ * @param {string} options.file
  * @param {string[]|string} options.releaseTypeScope
  * @returns {string}
  */
-const commitFiles = (version, { contextPath, releaseTypeScope } = OPTIONS) => {
+const commitFiles = (version, { contextPath, file, releaseTypeScope } = OPTIONS) => {
+  const filePath = join(contextPath, file);
```

Comment:
optional:
review centralizing the filePath generation, silly to write it multiple times - once here, and again in `files.js`